internal/linker/index.js

```diff
@@ -47,16 +47,33 @@ Include as much of the build output as you can without disclosing anything confi
     function mkdirp(p) {
         if (!fs.existsSync(p)) {
             mkdirp(path.dirname(p));
-            fs.mkdirSync(p);
+            log_verbose(`mkdir( ${p} )`);
+            try {
+                fs.mkdirSync(p);
+            }
+            catch (e) {
+                if (e.code !== 'EEXIST') {
+                    // can happen if path being created exists via a symlink
```

Comment:
because of a race condition creating the directory? or because fs.existsSync doesn't detect symlinks?

Reply:
the latter; fs.exists doesn't seem to look into directory symlinks which was surprising; fs.existsSync would return false but then mkdir would fail as the target directory already exists